src/lib/variants.ts

```diff
@@ -344,23 +359,14 @@ export const findTestAndVariant = (
 
     if (test) {
         const variant = selectVariant(test, targeting.mvtId || 1);
-        // Never show reminder feature to supporters
-        const showReminderFields = isSupporter
-            ? undefined
-            : getReminderFields(variant.showReminderFields);
-
-        const variantWithReminder: Variant = {
-            ...variant,
-            showReminderFields,
-        };
```

Comment:
We moves this from here to `buildEpicData` as we also want to run this logic when we force ourselves into a variant